live/README.md

```diff
@@ -0,0 +1,240 @@
+    - [SSH Server](#ssh-server)
+    - [Autologin](#autologin)
+    - [Firefox Profile](#firefox-profile)
+    - [Dracut menu](#dracut-menu)
+    - [Avahi/mDNS](#avahimdns)
+      - [The Default Hostname](#the-default-hostname)
+      - [Service Advertisement](#service-advertisement)
+    - [The Default Cockpit/Agama TCP Port](#the-default-cockpitagama-tcp-port)
+    - [Autoinstallation Support](#autoinstallation-support)
+    - [Firmware Cleanup](#firmware-cleanup)
+
+---
+
+## Layout
+
+This directory contains a set of files that are used to build the Agama Live ISO
+image.
+
+- [src](src) subdirectory contains all source files which are copied unmodified
+  to the OBS project
+- [root](root) subdirectory contains files which are added to the Live ISO root
+  system (inside the squashfs image)
+- [root-ALP-PXE](root-ALP-PXE) subdirectory contains specific files for the ALP
+  image used for the PXE boot, see a separate [PXE documentation](PXE.md) for
+  more details about the PXE boot
+- [config-cdroot](config-cdroot) subdirectory contains file which are copied to
+  the uncompressed root of the ISO image, the files can be accessed just by
+  mounting the ISO file or the DVD medium
+
+## Building the Sources
+
+To build the sources for OBS just run the
+
+```shell
+make
+```
+
+command. This will save the built source files into the `dist` subdirectory.
+
+To start from scratch run
+
+```shell
+make clean
+```
+
+or just simply delete the `dist` subdirectory.
+
+## Building the ISO Image
+
+To build the ISO locally run the
+
+```shell
+make build
+```
+
+command. The built ISO image is saved to the `/var/tmp/build-root` directory,
+see the end of the build for output for the exact ISO file name.
+
+For building an ISO image you need a lot of free space at the `/var` partition.
+Make sure there is at least 25GiB free space otherwise the build will
+fail.
+
+### Build Options
+
+By default this will build the openSUSE image. If you want to build
+another image then run
+
+```shell
+make build FLAVOR=<flavor>
+# for building the ALP flavor:
+make build FLAVOR=ALP
+```
+
+See the [_multibuild](src/_multibuild) file for the list of available build
+flavors.
+
+By default it will use the [systemsmanagement:Agama:Staging](
+https://build.opensuse.org/project/show/systemsmanagement:Agama:Staging) OBS
+project. If you want to build using another project, like your fork, then delete
+the `dist` directory and checkout the OBS project manually and run the build:
+
+```shell
+rm -rf dist
+# replace <USER> with your OBS account name
+osc co -o dist home:<USER>:branches:systemsmanagement:Agama:Staging agama-live
+make build
+```
+
+## Image Definition
+
+The [KIWI](https://github.com/OSInside/kiwi) image builder is used by OBS to
+build the Live ISO. See the [Kiwi documentation](
+https://osinside.github.io/kiwi/index.html) for more details about the build
+workflow and the `.kiwi` file format.
+
+### Kiwi Files
```

Comment:
np: I would say the name of the project is *KIWI* (all capital letters) (see [KIWI homepage](https://osinside.github.io/kiwi/)).